### ISSUE TITLE
Cloudflare Workers Vitest integration Broken by this package

### ISSUE THREAD

**Issue by andrew-pyle** [cloudflare/itty-router-openapi#141]
The Cloudflare Workers recommended testing strategy, the [Vitest Integration](https://developers.cloudflare.com/workers/testing/vitest-integration/), is not compatible with this package. It's due to a dependency, [@asteasolutions/zod-to-openapi](https://github.com/asteasolutions/zod-to-openapi). This package depends on `v5` of  [@asteasolutions/zod-to-openapi](https://github.com/asteasolutions/zod-to-openapi), which was not published in ESM format. Vitest only works with ESM code.

The dependency has already been fixed in `v7`. Can this package be updated to use the `v7` of [@asteasolutions/zod-to-openapi](https://github.com/asteasolutions/zod-to-openapi)?

References

- https://github.com/asteasolutions/zod-to-openapi/issues/219
- https://github.com/asteasolutions/zod-to-openapi/releases/tag/v7.0.0

**Comment by andrew-pyle** [comment]
I've overriden `@asteasolutions/zod-to-openapi` in my app's `package.json` to use `v7` with no ill effects so far.

In package.json:
```json
{
  "dependencies": {
    "@cloudflare/itty-router-openapi": "^1.1.0",
    "zod": "^3.22.4"
  },
  "overrides": {
    "@cloudflare/itty-router-openapi": {
      "@asteasolutions/zod-to-openapi": "^7.0.0"
    }
  }
}
```

**Comment by andrew-pyle** [comment]
Duplicate of #144. Resolved in v1.1.1.
